app/src/main/java/fr/gaulupeau/apps/Poche/service/IntentServiceBase.java

```diff
@@ -75,6 +75,12 @@ protected ActionResult processException(Exception e, String scope) {
             }
             // IOExceptions in most cases mean temporary error,
             // in some cases may mean that the action was completed anyway.
+        } else if(e instanceof IllegalArgumentException) {
```

Comment:
Maybe move the ternary directly into the `else if(…)` condition?
Something like this:
```suggestion
        } else if(e instanceof IllegalArgumentException && !getSettings().isConfigurationOk()) {
           result.setErrorType(ActionResult.ErrorType.INCORRECT_CONFIGURATION);
```

Reply:
Thanks, I can do this. However, I don't know if the `IllegalArgumentException` occurs if and only if the server is not configured. Can you help me with this? Because then the settings check would be unnecessary.

Reply:
`IllegalArgumentException` is too broad - the settings check is necessary. And I endorse the suggested change.

Reply:
Ok, I changed it!